geometry_msgs/Point normal
geometry_msgs/Point center
float64 radius
float64[49] surfel_cov
